Track Buddy/MotionManager.swift

```diff
@@ -12,32 +12,57 @@ import Collections
 import CoreGraphics
 
 class MotionManager: ObservableObject {
+    private enum Parameters {
+        static let deviceMotionUpdateInterval: TimeInterval = 1/100
+        static let graphUpdateInterval = RunLoop.SchedulerTimeType.Stride(1/15)
+        static var pointStorageLimit: Int {
+            Int(1 / graphUpdateInterval.magnitude * 3)
+        } // final number after multiplier == number of seconds retained for tracer line
+    }
+    
+    // Expose graph update interval to Views that need it
```

Comment:
This comment is not very useful. Should be more descriptive of what it actually is, not where it's used externally (as that can always change).

For comments above public / internal properties, get used to doing doc comments (///). These show up when you option-click a property or function. Write as if you're producing a public API / framework. 

(Definitely earns points in interviews btw, whether take-home or pairing)